service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/PopulateStandardDirectionsHandler.java

```diff
@@ -117,20 +99,25 @@ public void populateStandardDirections(PopulateStandardDirectionsEvent event) th
         return caseDetails.getData();
     }
 
-    private LocalDateTime getCompleteByDate(CaseData caseData, DirectionConfiguration direction) {
-        LocalDateTime completeBy = null;
+    private LocalDateTime getFirstHearingStartDate(List<Element<HearingBooking>> hearings) {
+        return hearingBookingService.getFirstHearing(hearings)
+            .map(HearingBooking::getStartDate)
+            .orElse(null);
```

Comment:
That or else null throws NullPointerExcpetion later (line 121). Would be much better if it fails here

Reply:
hmm so this is a weird one, because we are in submitted callback I can't block anything, do you think just throwing an exception is fine?

Reply:
Yes, throwing exception in this line would be better than delaying to line 121.

Reply:
@akomisarek ☝️